src/main/webapp/jsp/adminEmailLog.jsp

```diff
@@ -7,35 +7,35 @@
 <%@ taglib tagdir="/WEB-INF/tags/admin/email/log" prefix="adminEmailLog" %>
 
 <c:set var="jsIncludes">
-    <script type="text/javascript" src="<%= FrontEndLibrary.JQUERY_HIGHLIGHT %>"></script>
-    <script type="text/javascript" src="/js/adminEmailLog.js"></script>
+  <script type="text/javascript" src="<%= FrontEndLibrary.JQUERY_HIGHLIGHT %>"></script>
+  <script type="text/javascript" src="/js/adminEmailLog.js"></script>
 </c:set>
 
 <ta:adminPage bodyTitle="Admin Email Log" pageTitle="TEAMMATES - Administrator" jsIncludes="${jsIncludes}">
-    <adminEmailLog:filterPanel filterQuery="${data.filterQuery}" queryKeywordsForReceiver="${data.queryKeywordsForReceiver}"
-                               queryKeywordsForSubject="${data.queryKeywordsForSubject}" queryKeywordsForContent="${data.queryKeywordsForContent}"/>
+  <adminEmailLog:filterPanel filterQuery="${data.filterQuery}" queryKeywordsForReceiver="${data.queryKeywordsForReceiver}"
+                 queryKeywordsForSubject="${data.queryKeywordsForSubject}" queryKeywordsForContent="${data.queryKeywordsForContent}"/>
 
-    <%-- this form is used to store parameters for ajaxloader only --%>
-    <form id="ajaxLoaderDataForm">
-        <input type="hidden" name="offset" value="">
-        <%-- This parameter determines whether the logs with requests contained in "excludedLogRequestURIs"
-             in AdminActivityLogPageData should be shown. Use "?all=true" in URL to show all logs. This will keep showing all
-             logs despite any action or change in the page unless the page is reloaded with "?all=false"
-             or simply reloaded with this parameter omitted. --%>
+  <%-- this form is used to store parameters for ajaxloader only --%>
+  <form id="ajaxLoaderDataForm">
+    <input type="hidden" name="offset" value="">
+    <%-- This parameter determines whether the logs with requests contained in "excludedLogRequestURIs"
+       in AdminActivityLogPageData should be shown. Use "?all=true" in URL to show all logs. This will keep showing all
+       logs despite any action or change in the page unless the page is reloaded with "?all=false"
+       or simply reloaded with this parameter omitted. --%>
```

Comment:
From http://www.oracle.com/technetwork/articles/javase/code-convention-138726.html which is linked in [our JSP coding standard](https://docs.google.com/document/d/14bXfdveXvoIaPBYpL19m4PK6oPabSnnoawj6OGjOzD4/pub?embedded=true):
```
<%--
  - This parameter determines...
  - in AdminActivityLogPageData...
  - logs despite any action...
  - or simply reloaded with this parameter omitted.
  --%>
```